stablehlo/conversions/tosa/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022 OpenXLA Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
+
+add_custom_target(check-stablehlo-tosa)
```

Comment:
Let's add this target to CI? Alternatively, we could use `check-stablehlo-tests` here, which is already run in CI.

Reply:
I did the latter as it seems more bullet proof (and already in email). Again fine to redirect if we find the cost to dev and testing is high.